include/lsst/jointcal/Frame.h

```diff
@@ -28,8 +28,8 @@ class Frame {
 
   //! this one is dangerous: you may swap the 2 middle arguments.
   //! Prefer next one
-  Frame(const double &xMin, const double &yMin,
-	const double &xMax, const double &yMax);
+  Frame(double xMin, double yMin,
+	double xMax, double yMax);
```

Comment:
If you meant "make it one line", then done. If you meant something else, please clarify.